html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Profile.pm

```diff
@@ -16,16 +16,19 @@ use strict;
 use warnings;
```

Comment:
There are still subroutines that need naming cleanup 
_makeDefaultFilePath
copyDefaultFiles